exporters/otlp/src/main/java/io/opentelemetry/exporters/otlp/SpanAdapter.java

```diff
@@ -169,12 +171,16 @@ static Span toProtoSpan(SpanData spanData) {
   }
 
   static Status toStatusProto(SpanData.Status status) {
-    // todo: Update this when the proto definitions are updated to include UNSET and ERROR
     Status.StatusCode protoStatusCode = Status.StatusCode.STATUS_CODE_OK;
```

Comment:
I think there is a mistake here: For UNSET, we should use UNSET, not OK.

Reply:
Thanks. Can you review again?

Reply:
Thanks, looks good now!